circle.yml

```diff
@@ -382,7 +380,8 @@ jobs:
       - run: npm run check:markdown
 
   build-base-image:
-    machine: true
+    machine:
+      image: ubuntu-2004:202111-02
```

Comment:
Why is this needed on this PR? Is this out of scope? It looks like the initial commit built successfully.